Hangman-Game/main.py

```diff
@@ -0,0 +1,149 @@
+import pygame, math, random
+
+#setup display
+pygame.init() 
+WIDTH, HEIGHT= 800,500 
+win = pygame.display.set_mode((WIDTH,HEIGHT)) 
+pygame.display.set_caption("Hangman Game") 
+
+
+#button variables
+radius = 20
+gap= 15
+letters = []
+startx= round((WIDTH - (radius * 2 + gap) * 13) / 2) 
+starty= 400 
+
+A= 65
+
+for i in range(26): 
+  x= startx + gap * 2+ ((radius * 2 + gap)* (i%13))
+  y= starty + ((i//13) * (gap + radius * 2))
+  letters.append([x, y, chr(A+i), True]) 
+
+
+
+#load images
+images = []
+for i in range(7):
+  img = pygame.image.load("hangman" + str(i) + ".png")
+  images.append(img)
+
+
+
+
+#game variables
+hangman_status = 0 
+list_of_words= ["abruptly","avenue","awkward","azure","galaxy","gossip","icebox","injury","ivory","ivy","jackpot","jaundice","joyful","juicy","jukebox","jumbo","kiwifruit","matrix","microwave","nightclub","nowadays","oxidize","oxygen","peekaboo","pixel","pneumonia","puppy","puzzling","queue","quizzes","quorum","rhythm","rickshaw","scratch","staff","strengths","stretch","subway","syndrome","thumbscrew","transcript","transplant","twelfth","unknown","unworthy","unzip","uptown","vodka","vortex","walkway","wave","wavy","whiskey","whizzing","wizard","wristwatch","xylophone","yachtsman", "youthful","yummy","zigzag" ,"zodiac" ,"zombie"]
```

Comment:
instead of hardcoding these words, create a separate text file containing all these words and read that file in the program. It will make updation of words easier.